storage/indexer/src/db_indexer.rs

```diff
@@ -441,6 +526,60 @@ impl DBIndexer {
         Ok(version)
     }
 
+    pub fn get_state_value_for_resource(
+        &self,
+        address: &AccountAddress,
+        struct_tag: &StructTag,
+    ) -> Result<Option<StateValue>> {
+        let state_view = self
+            .main_db_reader
+            .latest_state_checkpoint_view()
+            .expect("Failed to get state view");
+
+        //let struct_tag = StructTag::from_str(struct_tag_str)?;
```

Comment:
?

Reply:
Thank you for finding it. This is by mistake. I just removed the line.